.github/ISSUE_TEMPLATE/1-bug-report.yaml

```diff
@@ -0,0 +1,31 @@
+name: Bug Report
+description: Report a bug in the Auditor Framework
+
+body:
+  - type: dropdown
+    id: is-regression
+    attributes:
+      label: Is this a regression?
+      options:
+        - 'Yes'
+        - 'No'
+    validations:
+      required: true
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+    validations:
+      required: true
+
+  - type: textarea
+    id: exception-or-error
+    attributes:
+      label: Please provide the exception or error you saw
+      render: true
+
+  - type: textarea
+    id: other
+    attributes:
+      label: Anything else?
```

Comment:
It's better to end .yaml file with new line.  Same for others yaml in the file

Reply:
Addressed